### PR TITLE
Prepare for v0.8.1 release by bumping the version and changelog

### PR DESCRIPTION
**Pull request by sgnn7**
We are preparing to release the current bundle of features as 0.8.1 so
we incremented the version in both version.go and the changelog.

### CODE REVIEW

**Comment by izgeri** [comment]
can we include in this release a small update to CONTRIBUTING to add instructions for how you generate a release? there's nothing in there on this yet, and as you're about to go through it I think it'd be really helpful

it would be great if this could also reference updating the homebrew-tools with the new release info, if that's not automated.

**Comment by sgnn7** [comment]
@izgeri Updated. CR ready again.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,3 +28,34 @@ To build versions for Linux, OSX and Windows:
 ```
 
 Binaries will be placed in `output/`.
+
+### Releasing
+
+The collowing checklist should be followed when creating a release:
+
+- [ ] Open a PR with the following changes for `master` branch and wait for it to be merged:
+  - [ ] Bump release version in [`pkg/summon/versino.go`](pkg/version/summon.go).
```
Comment (izgeri) [inline CONTRIBUTING.md]:
```suggestion
  - [ ] Bump release version in [`pkg/summon/version.go`](pkg/summon/version.go).
```

Comment (sgnn7) [reply CONTRIBUTING.md]: Fixed. CR ready again.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,3 +28,34 @@ To build versions for Linux, OSX and Windows:
 ```
 
 Binaries will be placed in `output/`.
+
+### Releasing
+
+The following checklist should be followed when creating a release:
+
+- [ ] Open a PR with the following changes for `master` branch and wait for it to be merged:
+  - [ ] Bump release version in [`pkg/summon/version.go`](pkg/version/summon.go).
```
Comment (izgeri) [inline CONTRIBUTING.md]: this is still broken?
```suggestion
  - [ ] Bump release version in [`pkg/summon/version.go`](pkg/summon/version.go).
```

**Pull request merged**